mw-webapp/.eslintrc

```diff
@@ -18,8 +18,9 @@
     // "project": ["**/tsconfig.json", "**/vite.config.ts"] // путь для сопоставления файлов tsconfig относительно parserOptions.tsconfigRootDir, ​​или текущего рабочего каталога, если он не указан
     "project": true // указывает на поиск ближайшего tsconfig.json для каждого исходного файла
   },
-  "plugins": ["react", "@typescript-eslint", "simple-import-sort"],
+  "plugins": ["react", "@typescript-eslint", "simple-import-sort", "import"],
   "rules": {
+    "import/no-default-export": "off",
```

Comment:
Can't we use https://eslint.org/docs/latest/rules/no-restricted-exports instead of add new dependency?
If you think that we can't use it then explain please why?

Reply:
updated